chsdi/locale/de/LC_MESSAGES/chsdi.po

```diff
@@ -1442,18 +1442,288 @@ msgstr "Zählstelle"
 msgid "ch.astra.unfaelle-personenschaeden_alle"
 msgstr "Unfälle mit Personenschaden"
 
+msgid "ch.astra.unfaelle-personenschaeden_alle.accidentday_de"
+msgstr "Unfalltag"
+
+msgid "ch.astra.unfaelle-personenschaeden_alle.accidentday_fr"
+msgstr "Unfalltag"
+
+msgid "ch.astra.unfaelle-personenschaeden_alle.accidentday_it"
+msgstr "Unfalltag"
```

Comment:
Not possible to use one token here? 
`ch.astra.unfaelle-personenschaeden_alle.accidentday`